starknet_py/tests/e2e/conftest.py

```diff
@@ -194,6 +196,22 @@ def compiled_proxy(request) -> str:
     return (directory_with_contracts / request.param).read_text("utf-8")
 
 
+@pytest.fixture(
+    name="typed_data",
+    params=["typed_data_example.json", "typed_data_struct_array_example.json"],
+)
+def typed_data(request) -> TypedData:
+    file_name = getattr(request, "param")
+
+    directory = Path(os.path.dirname(__file__))
+    file_path = directory / "account" / file_name
+
+    with open(file_path, "r", encoding="utf-8") as file:
+        typed_data = json.load(file)
```

Comment:
I think you can do `file_path.read_text("utf-8")`, so you don't have to manage opening the file.